README.md

```diff
@@ -1,5 +1,106 @@
+[![CI](https://github.com/astronomer/astro-providers-databricks/actions/workflows/ci.yml/badge.svg)](https://github.com/astronomer/astro-providers-databricks)
+[![codecov](https://codecov.io/gh/astronomer/astro-providers-databricks/branch/main/graph/badge.svg?token=MI4SSE50Q6)](https://codecov.io/gh/astronomer/astro-providers-databricks)
 
-For up-to-date documentation, visit https://astronomer.github.io/astro-providers-databricks/
\ No newline at end of file
+
+**Astro Databricks** is an [Apache Airflow](https://github.com/apache/airflow) provider created by [Astronomer](https://www.astronomer.io/) for an **optimal Databricks experience**.  With the `DatabricksTaskGroup`, Astro Datatricks allows you to run from Databricks workflows without
+the need of running Jobs individually, which can result in **75% cost reduction**.
+
+## Prerequisites
+
+* Apache Airflow >= 2.2
+* Python >= 3.7
+* Databricks account
+* Previously created Databricks Notebooks
+
+## Install
+
+```shell
+pip install astro-providers-databricks
+```
+
+## Quickstart
+
+1. Use pre-existing or create two simple [Databricks Notebooks](https://docs.databricks.com/notebooks/). Their identifiers will be used in step (5). The original example DAG uses: 
+   * `Shared/Notebook_1`
+   * `Shared/Notebook_2`
+
+2. Generate a [Databricks Personal Token](https://docs.databricks.com/dev-tools/auth.html#databricks-personal-access-tokens). This will be used in step (6). 
+
+3. Ensure that your Airflow environment is set up correctly by running the following commands:
+
+    ```shell
+    export AIRFLOW_HOME=`pwd`
+    export AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES": "airflow\\.* astro_databricks\\.*"
+   
+    airflow db init
+    ```
+   
+4. [Create](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html) a Databricks Airflow connection (so Airflow can access Databricks using your credentials). This can be done by running the following command, replacing the login and password (with your access token):
+
+```shell
+airflow connections add 'databricks_conn' \
+    --conn-json '{
+        "conn_type": "databricks",
+        "login": "some.email@yourcompany.com",
+        "host": "https://dbc-c9390870-65ef.cloud.databricks.com/",
+        "password": "personal-access-token"
+    }'
+```
+
+5. Copy the following workflow into a file named `example_databricks_workflow.py` and add it to the `dags` directory of your Airflow project:
+   
+   https://github.com/astronomer/astro-providers-databricks/blob/45897543a5e34d446c84b3fbc4f6f7a3ed16cdf7/example_dags/example_databricks_workflow.py#L48-L101
+
+   Alternatively, you can download `example_databricks_workflow.py`
+   ```shell
+    curl -O https://raw.githubusercontent.com/astronomer/astro-providers-databricks/main/example_dags/example_databricks_workflow.py
```

Comment:
Update to the new `example_dags` directory

Reply:
We'll rever the `example_dags` change to move to the root directory, so it won't be necessary